### ISSUE TITLE
DioError[DioErrorType.DEFAULT]: Invlid argument(s)

### ISSUE THREAD

**Issue by zhangweiwei0326** [flutterchina/dio#604]
### New Issue Checklist

- [ ] I have searched for a similar issue in the [project](https://github.com/flutterchina/dio/issues) and found none

### Issue Info

| Info                           | Value                                                 |      |
| ------------------------------ | ----------------------------------------------------- | ---- |
| Platform Name                  | e.g. flutter / ios / android                          |      |
| Platform Version               | e.g.   13.2.3                              |      |
| Dio Version                    | e.g.  3.0.7                                |      |
| Android Studio / Xcode Version | e.g. Android Studio 3.3.2 / Xcode 10.2.1              |      |
| Repro rate                     | e.g. only once   |      |
| Repro with our demo prj        | e.g. does it happen with our demo project?            |      |
| Demo project link              | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

![WechatIMG1430](https://user-images.githubusercontent.com/5975616/70847086-15a32900-1e9b-11ea-8025-485a126eaa5f.jpeg)
